index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset='utf-8'>
+  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+  <title>Portfolio Skelenton</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
+  <link rel='stylesheet' type='text/css' media='screen' href='/css/styles.css'>
```

Comment:
- [ ] Good job :+1:  on working on this task. I noticed your application is not loading the styles correctly. This is a result of the wrong path to the CSS file. Kindly try to replace `href='/css/styles.css'` to `href='./css/styles.css'` 

Your Design without the styles        |
:---------------------------------------------:|
![e1](https://user-images.githubusercontent.com/39523312/147312583-02f2a3f6-78f0-44e5-ac91-7a20c325a909.png)